src/UI/InputField/InputField.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../styles/variables.scss';
+
+.input-group-UI {
+  position: relative;
+  margin-top: 20px;
+  .eye-icon {
+    position: absolute;
+    top: 21%;
+    right: 0;
+    margin-right: 8px;
+  }
+}
```

Comment:
Let's imagine that the element has a height of 100 pixels. Then 21% is 21 pixels. But what if we change the content of the element or its dimensions? 21% becomes another distance. Do not rely on percentages to position elements that must be located at a specific and precise distance from the borders of this element.
Better yet, look at the main page of Google, at the search bar. You can make a form with all the icons without resorting to using absolute positioning. Or, if you can't do without it, use the hack with 50% and transform by -50%:
```
top: 50%;
transform: translate(0,-50%);
```
Also, eye is not just svg element - this is a button.